aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -58,9 +58,9 @@ impl GroupWrite {
     pub fn new(
         mut metadata_op: WriteOp,
         group_size: u64,
-        inner_ops: HashMap<StructTag, WriteOp>,
+        inner_ops: HashMap<StructTag, (WriteOp, Option<Arc<MoveTypeLayout>>)>,
```

Comment:
I suggest either cherrypicking the sev changes or merging main to aggregator branch so we don't accidentally merge change it back to hasmap.